lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua

```diff
@@ -110,11 +110,11 @@ e2function void entity:npcSetGlobalSquad( string squadIn )
     this:SetKeyValue( "squadname", squadIn )
 end
 
-e2function entity npcCreate( string npcClass, vector position )
+e2function entity npcCreate( string npcClass, vector positionIn )
     if not hasAccess( self.player, 1 ) then return NULL end
+    position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```

Comment:
If you really prefer `positionIn` over `position`, then this needs to be a local variable

```suggestion
    local position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```